opentelemetry-tracing/opentelemetry-java-tracing/spring-boot-kotlin/pom.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.1.5.RELEASE</version>
+        <relativePath/>
+    </parent>
+    <groupId>com.splunk.tracing.otel.examples</groupId>
+    <artifactId>kotlin-spring-demo</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>Wishlist</name>
+    <description>Example Spring Boot application in Kotlin</description>
+
+    <properties>
+        <java.version>1.8</java.version>
+        <kotlin.version>1.2.71</kotlin.version>
```

Comment:
That is ancient kotlin version, can we upgrade?

Reply:
Set the newest one.